lib/Service/LockService.php

```diff
@@ -344,13 +344,7 @@ public function injectMetadata(FileLock $lock): FileLock {
 			$displayName = $this->getAppName($lock->getOwner()) ?? null;
 		}
 		if ($lock->getType() === ILock::TYPE_TOKEN) {
-			if (!method_exists($this->userManager, 'getDisplayName')) {
-				// TODO: Remove once 25 is the minimum supported version
-				$user = $this->userManager->get($lock->getOwner());
-				$displayName = $user ? $user->getDisplayName(): $lock->getDisplayName();
-			} else {
-				$displayName = $this->userManager->getDisplayName($lock->getOwner()) ?? $lock->getDisplayName();
-			}
+			$displayName = $this->userManager->getDisplayName($lock->getOwner()) ?? $lock->getDisplayName();
```

Comment:
The same TODO is on line 337 as well.
See e35d7ea0ad528e20fca57c32e1b401cb7262970a
See also https://github.com/nextcloud/files_lock/blob/main/lib/Tools/Db/ExtendedQueryBuilder.php#L62

Reply:
Thanks, also adjusted